frontend/fingertips-frontend/components/pages/chart/chart.test.tsx

```diff
@@ -47,3 +54,19 @@ describe('Page structure', () => {
     });
   });
 });
+
+it('should render the PopulationPyramid component when Population data are provided', () => {
+  render(
+    <Chart
+      data={mockHealthData[1]}
+      preparedPopulationData={{
+        dataForSelectedArea: mockPopulationData,
+        dataForEngland: undefined,
+        dataForBaseline: undefined,
+      }}
+    />
+  );
+
+  const PyramidPlot = screen.getByTestId('populationPyramid-component');
```

Comment:
change to `populationPyramid` - we should avoid naming something that is not a class with an uppercase first letter.